assets/config.js

```diff
@@ -12,7 +12,8 @@ const Config = {
 	wpDomain: "https://wecount.inclusivedesign.ca",
 	loadDbName: "starter_wp",
 	apiBase: "/wp-json/wp/v2/",
-	contactEmail: "wecount@inclusivedesign.ca"
+	contactEmail: "wecount@inclusivedesign.ca",
+	socialMedia: [{ title: "Twitter", href: "https://twitter.com/wecount" }]
```

Comment:
@TeddTech I'd actually leave this as an empty array.

Reply:
Like an empty string? `""`

Reply:
No, an empty array: `[]`. We don't have any social media so we should not provide any.

Reply:
Oh okay. Thought you were talking about `contactEmail`

Reply:
It is completed now